lib/documentLoaders/xhr.js

```diff
@@ -30,7 +34,7 @@ module.exports = ({
   const queue = new RequestQueue();
   return queue.wrapLoader(loader);
 
-  async function loader(url) {
+  async function loader(url, options) {
```

Comment:
There don't seem to be standalone XHR documentLoader tests, as there are node tests.